relevanceai/reports/cluster/report.py

```diff
@@ -551,13 +570,15 @@ def _store_basic_centroid_stats(self, overall_report):
         if self.has_centers():
             centroids = self.get_centers()
             overall_report["centroids"] = centroids
-            overall_report["centroids_distance_matrix"] = pairwise_distances(
-                centroids, metric="euclidean"
+            overall_report["centroids_distance_matrix"] = np.around(
+                pairwise_distances(centroids, metric="euclidean"), 3
```

Comment:
Why does this place use 3 instead of DECIMAL_PLACES

Reply:
missed it in crl-f